acceptance_tests/features/add_a_survey.feature

```diff
@@ -16,7 +25,7 @@ Feature: Add a survey
 
   @fixture.setup.data.with.unenrolled.respondent.user
   @us334-addSurvey_s02
-  Scenario: Enter the enrolment code
+  Scenario: Enter the enrolment codegit
```

Comment:
I don't think you meant to commit this change

Reply:
correct, haha